site/src/examples/slider/WithFormFieldRange.tsx

```diff
@@ -0,0 +1,19 @@
+import { FormField, FormFieldLabel, StackLayout } from "@salt-ds/core";
+import { RangeSlider } from "@salt-ds/lab";
+import type { ReactElement } from "react";
+
+export const WithFormFieldRange = (): ReactElement => {
+  return (
+    <StackLayout style={{ width: "400px" }}>
+      <FormField labelPlacement="left">
+        <FormFieldLabel>Field Label</FormFieldLabel>
+        <RangeSlider min={0} max={50} step={10} />
+      </FormField>
+
+      <FormField style={{ width: "400px" }}>
```

Comment:
Maybe ask Design if this is a good representation of formfield labels?
As we put two sliders together with different label placements, it makes the "left" alignment look weird due to the size of the slider (400px). 
ie. the gap is rather large between label and component